elab_bridge/tests/test_server_interface.py

```diff
@@ -7,12 +7,23 @@
 def test_upload_template(initialize_test_dir):
     template_file = test_directory / 'testfiles_elab' / 'template.json'
 
-    res, http_stat_code = upload_template(server_config_json=SERVER_CONFIG_YAML, template_file=template_file, template_title='Testproject')
+    res, http_stat_code = upload_template(server_config_json=SERVER_CONFIG_YAML, template_file=template_file,
+                                          template_title='Testproject')
 
     # 200 is for creation of a template with metadata / 201 is for creation of a template without metadata
     assert http_stat_code == 200 or http_stat_code == 201
 
 
+def test_upload_experiment(initialize_test_dir):
+    template_file = test_directory / 'testfiles_elab' / 'template.json'
+
+    res, http_stat_code = upload_experiment(server_config_json=SERVER_CONFIG_YAML, experiment_file=template_file,
+                                            experiment_title='Test_upload_experiment')
```

Comment:
Maybe better for consistency with the template_title:
```suggestion
                                            experiment_title='TestExperiment')
```